waterlog/src/Components/LeakCosts.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
```

Comment:
make double quotes single quotes

Reply:
does not matter just needs to be consistent

Reply:
Yeah choose one and stick to it.

Reply:
made quotes single quotes.

Reply:
File deleted